specification/compute/resource-manager/Microsoft.Compute/2015-06-15/compute.json

```diff
@@ -50,7 +50,7 @@
             "description": "The name of the resource group."
           },
           {
-            "name": "name",
+            "name": "availabilitySetName",
```

Comment:
Please just port https://github.com/Azure/azure-rest-api-specs/pull/1972, which use a `x-ms-client-name` to avoid breaking change

Reply:
@hyonholee please refer https://github.com/veronicagg/azure-rest-api-specs/blob/e9274c90a844e96ab19e3cb562010fac5230379d/specification/compute/resource-manager/Microsoft.Compute/2015-06-15/compute.json#L54